packager/config.js

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+/* eslint-disable no-console */
+const modulePaths = require('./modulePaths');
+const resolve = require('path').resolve;
+const fs = require('fs');
+
+const platformRegex = /\.(android\.js|ios\.js)/g;
+const modulesRegex = /\/(node_modules)/;
+
+const config = {
```

Comment:
Can we add some comments to document what this code is accomplishing

Reply:
sure I can do that